src/Settings/Builders/RunTest.hs

```diff
@@ -17,6 +16,7 @@ oneZero lbl True = lbl ++ "=1"
 stringToBool :: String -> Bool
 stringToBool "YES"  = True
 stringToBool "NO"   = False
+stringToBool _      = error "Cannot parse string"
```

Comment:
Let's rename this function to something more obvious, e.g. `parseYesNo`? Also I would prefer to make it type safe by returning `Maybe Bool` and handling possible failures at the use site.

Also, note that we already have something like this here:

https://github.com/snowleopard/hadrian/blob/master/src/Oracles/Flag.hs#L39-L40

As you can see, handling such errors at the use site allows for more informative error messages.